app/gui/CHANGELOG.md

```diff
@@ -2,11 +2,15 @@
 
 #### Visual Environment
 
+- [Reimplemented Welcome Screen.][3167]. Welcome Screen is now integrated much
+  tightly into the codebase and it will make further improvements a lot easier
+  for developers.
```

Comment:
@wdanilo correct me if I'm wrong, but we usually don't put info for developers in Changelog?

Reply:
Yeah, I was a little hesitant about this, but decided to add it because the change is noticeable for users: loading animation is now visible before Welcome Screen, not after; and the mouse cursor is replaced with Enso's circular cursor. I'll remove it if you think it doesn't need to be mentioned in the changelog.

Reply:
If there is something noticeable for the user, that could be what is mentioned and highlighted in the changelog. They are the target audience of the changelog.

Reply:
I removed this changelog entry. Changes are noticeable, but not significant for the users.